spec/factories/child_case_cycles.rb

```diff
@@ -0,0 +1,41 @@
+FactoryBot.define do
+  factory :child_case_cycle do
+    user
+    child
+    subsidy_rule
+    case_cycle
+    part_days_allowed { 1 }
+    full_days_allowed { 1 }
```

Comment:
I wasn't sure of the reasonable defaults for these attributes. Let me know what values we should use here.

Reply:
I'd do a random number between 10 and 20 maybe